### PR TITLE
Feat: k8 manifests and helm updates

### PR DESCRIPTION
**Pull request by ChrisRousey**
Added python statefulset execution to k8 manifests
added flag to helmchart to remove helm-specific labels

### CODE REVIEW

File: k8s-manifests/minikube-any-namespace.yaml [hunk 1]
```diff
@@ -201,6 +222,66 @@ spec:
...
+            httpGet:
+              path: /health
+              port: 50000
+              scheme: HTTP
+            initialDelaySeconds: 0
+            periodSeconds: 1
+            successThreshold: 1
+            timeoutSeconds: 10
+          readinessProbe:
+            failureThreshold: 3
```
Comment (flippingbits) [inline k8s-manifests/minikube-any-namespace.yaml:269]: We need to bump the failureThreshold to `10`, the Python Runner typically takes a couple of seconds to start.

File: k8s-manifests/minikube-any-namespace.yaml [hunk 2]
```diff
@@ -201,6 +222,66 @@ spec:
...
+    spec:
+      containers:
+        - name: python-runner
+          image: "datacater/python-runner:nightly-20230105"
+          imagePullPolicy: IfNotPresent
+          ports:
+            - name: http
+              containerPort: 50000
+          livenessProbe:
+            failureThreshold: 3
```
Comment (flippingbits) [inline k8s-manifests/minikube-any-namespace.yaml:259]: We might want to adjust the failureTreshold to `10` as well here, similar to the readinessProbe.

File: k8s-manifests/minikube-with-postgres-ns-default.yaml [hunk 3]
```diff
@@ -210,7 +291,7 @@ spec:
             - name: QUARKUS_DATASOURCE_REACTIVE_URL
               value: "postgresql://postgres.default.svc.cluster.local:5432/postgres"
             - name: DATACATER_PYTHONRUNNER_IMAGE_NAMESPACE
-              value: default
+              value: datacater
```
Comment (flippingbits) [inline k8s-manifests/minikube-with-postgres-ns-default.yaml:294]: We should keep `default` here.

File: k8s-manifests/minikube-with-postgres-ns-default.yaml [hunk 4]
```diff
@@ -173,6 +194,66 @@ spec:
...
+            httpGet:
+              path: /health
+              port: 50000
+              scheme: HTTP
+            initialDelaySeconds: 0
+            periodSeconds: 1
+            successThreshold: 1
+            timeoutSeconds: 10
+          readinessProbe:
+            failureThreshold: 3
```
Comment (flippingbits) [inline k8s-manifests/minikube-with-postgres-ns-default.yaml:241]: Please bump it to `10`.

File: k8s-manifests/minikube-with-postgres-ns-default.yaml [hunk 5]
```diff
@@ -173,6 +194,66 @@ spec:
...
+    spec:
+      containers:
+        - name: python-runner
+          image: "datacater/python-runner:nightly-20230105"
+          imagePullPolicy: IfNotPresent
+          ports:
+            - name: http
+              containerPort: 50000
+          livenessProbe:
+            failureThreshold: 3
```
Comment (flippingbits) [inline k8s-manifests/minikube-with-postgres-ns-default.yaml:231]: Please bump it to `10`.

**Review by flippingbits — CHANGES_REQUESTED** [review]
Thanks! Left a couple of comments :)

**Review by HknLof — APPROVED** [review]

**Review by flippingbits — APPROVED** [review]

**Pull request merged**